### PR TITLE
feat: debug menu and fix android build, localization, some type check, add fonts

### PR DESCRIPTION
**Pull request by ThinhKimVo**

### CODE REVIEW

File: template/src/components/DebugMenu.tsx [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+    ],
+    [appName, buildNumber, deviceId],
+  )
+
+  return (
+    <View style={styles.section}>
+      <Text style={styles.h3}>{localize('debug.info')}</Text>
+      <View style={styles.content}>
+        {infos.map(({title, description}) => (
+          <InfoMenuRow style={styles.infoMenu} title={title} description={description} />
```
Comment (loc-nguyenthien) [inline template/src/components/DebugMenu.tsx]: Missing `key` prop

File: template/src/components/DebugMenu.tsx [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+  const currentApiUrl = useSelector(getApiUrl)
+
+  const [bottomSheetType, setBottomSheetType] = useState<string>()
+
+  const dimensions = useWindowDimensions()
+
+  const bottomSheetRef = useRef<BottomSheetMethods>(null)
+
+  const snapPoints = useMemo(() => ['95%'], [])
```
Comment (loc-nguyenthien) [inline template/src/components/DebugMenu.tsx]: Move this to a constants outside of component, so we don't have to use `useMemo`

File: template/src/components/DebugMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+        handleClosePress()
+      }
+      return (
+        <Pressable disabled={isActive} style={styles.flatListItem} onPress={onPress}>
+          <Text
+            style={[
+              styles.flatListItemTitle,
+              isActive
+                ? {color: colors.primary, fontWeight: 'bold'}
+                : {color: colors.black, fontWeight: 'normal'},
```
Comment (loc-nguyenthien) [inline template/src/components/DebugMenu.tsx]: Should avoid in-line styles

File: template/src/components/InfoMenu.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  action,
+  horizontal,
+  children,
+}) => (
+  <View style={[styles.container, style, horizontal && styles.row]}>
+    <View style={styles.titleWrapper}>
+      <Text style={titleStyle}>{title}</Text>
+      {action}
+    </View>
+    <Text style={[{color: colors.gray}, descriptionStyle]}>{description}</Text>
```
Comment (loc-nguyenthien) [inline template/src/components/InfoMenu.tsx]: avoid in-line styles

**Review by loc-nguyenthien — CHANGES_REQUESTED** [review]

**Review by loc-nguyenthien — APPROVED** [review]

**Review by hangnguyensaigontech — APPROVED** [review]

**Review by anhle10051996 — APPROVED** [review]

**Review by loido — APPROVED** [review]

**Pull request merged**